src/main/scala/scorex/core/transaction/state/StateFeature.scala

```diff
@@ -8,10 +8,6 @@ import scala.util.Try
 trait StateFeature
 
 trait TransactionValidation extends StateFeature {
-  def isValid(tx: ErgoTransaction): Boolean = validate(tx).isSuccess
-
-  def filterValid(txs: Seq[ErgoTransaction]): Seq[ErgoTransaction] = txs.filter(isValid)
-
   def validate(tx: ErgoTransaction): Try[Unit]
```

Comment:
Please write ScalaDocs for the class and the method.

Reply:
Done in 4.0.17 branch